ROADMAP.md

```diff
@@ -246,7 +241,58 @@ Any dates listed below and the specific issues that will ship in a given milesto
   - Cert Manager [crossplane/provider-aws#171](https://github.com/crossplane/provider-aws/issues/171)
   - DNS [crossplane/provider-aws#172](https://github.com/crossplane/provider-aws/issues/172)
 
-* Basic versioning and upgrade support [#1334](https://github.com/crossplane/crossplane/issues/1334)
+
+* Docs overhaul (part 3/3) - https://crossplane.io/docs
+  * Documentation (and diagrams) about data model in Crossplane (including both application and infrastructure)
+  * Updated docs sidebar
+  * Finish name refactoring e.g. StackInstall -> PackageInstall
+  * Add an experimental / deprecated section
+
+
+## [v0.12.0 infra composition enhancements towards v1beta1](https://github.com/crossplane/crossplane/releases/tag/v0.12.0)
+* Infrastructure Composition
+  * Default composition for a definition [crossplane#1471](https://github.com/crossplane/crossplane/issues/1471)
+  * Enforced composition for a definition [crossplane#1470](https://github.com/crossplane/crossplane/issues/1470)
+  * Enhanced testing [crossplane#1474](https://github.com/crossplane/crossplane/issues/1474)
+  * Deprecate resource claims and classes [crossplane#1479](https://github.com/crossplane/crossplane/issues/1479)
+
+* Package Manager
+  * Passing non-zero fsGroup in package deployments [crossplane#1577](https://github.com/crossplane/crossplane/pull/1577)
+
+* Provider AWS
+  * ACMPCA Certificate Authority [provider-aws#226](https://github.com/crossplane/provider-aws/pull/226) 
```

Comment:
Do we want to include this even though `provider-aws` has not released a version with this functionality?

Reply:
Good catch, moved ACMPCA to v0.13.